FlowCal/plot.py

```diff
@@ -917,8 +1000,8 @@ def density2d(data,
             cbar.ax.set_ylabel('Counts')
 
     # Set scale of axes
-    plt.gca().set_xscale(xscale)
-    plt.gca().set_yscale(yscale)
+    plt.gca().set_xscale(xscale, data=data_plot, channel=0)
+    plt.gca().set_yscale(yscale, data=data_plot, channel=1)
```

Comment:
> these keywords just get passed to the scale object's constructor, though, right?

Yes.

> do we know that these particular parameter names won't clobber/overload anything in the native matplotlib scale objects?

yep. check this: http://matplotlib.org/api/axes_api.html#matplotlib.axes.Axes.set_xscale